### ISSUE TITLE
Would it be possible to add more columns?

### ISSUE THREAD

**Issue by cjdduarte** [reuseman/flashcards-obsidian#21]
I use 3 columns in Anki:
1 - Front
2 - Back
3 - Extra

I would like to be able to configure a way to include information in these 3 columns.

**Comment by reuseman** [comment]
Hey @cjdduarte, totally understand your request, and it makes perfectly sense. But when I started the development of this plugin I did it in such way that it suited my workflow, so it's quite an opinionated plugin with few possibilities of customization. This allows me to have not a huge amount of maintenance and has a simple learning curve to get in for the average Joe. Probably [this](https://github.com/Pseudonium/Obsidian_to_Anki) alternative suits more of your advanced needs.

**Comment by reuseman** [comment]
Hey @cjdduarte , totally understand your request, and it makes perfectly sense.

But when I started the development of this plugin I did it in such way that it suited my workflow, so it's quite an opinionated plugin with few possibilities of customization. This allows me to have not a huge amount of maintenance and to have a simple learning curve and easiness for the average Joe.

So implementing what you ask me, will mean implementing a huge customization for the different type of decks that the user may provide and a different kind of customizable Regexp to parse the fields in the Obsidian file. I do not have the time to do this.

 Probably [this](https://github.com/Pseudonium/Obsidian_to_Anki) alternative suits more of your advanced needs.
